pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -8,6 +8,7 @@ with lib;
 
 { version, sha256, patches ? []
 , license ? lib.licenses.sspl
+, broken ? false
```

Comment:
Can we use something else to mark as deprecated, per https://www.mongodb.com/support-policy/lifecycles ?